MC/bin/o2dpg_sim_config.py

```diff
@@ -16,6 +12,12 @@ def add(cfg, flatconfig):
            d[sk] = flatconfig[entry]
            cfg[mk] = d
 
+    COLTYPEIR = args.col
+    if args.embedding==True:
+        COLTYPEIR = args.colBkg
+        add(config, {"ITSCATrackerParam.trackletsPerClusterLimit": 20,
+                     "ITSCATrackerParam.cellsPerClusterLimit": "20"})
+
```

Comment:
I don't fully understand this. Is this setting something specific to **all** embedding MCs? Even when we embedd pp into pp? Or is it rather to be applied whenever we have PbPb collisions (irrespective of embedding mode)?

Reply:
> I don't fully understand this. Is this setting something specific to **all** embedding MCs? Even when we embedd pp into pp? Or is it rather to be applied whenever we have PbPb collisions (irrespective of embedding mode)?

The latter you said

Reply:
in pp-pp embedding I don't expect you to need this

Reply:
but it wouldn't do any harm, meaning the settings can be applied in embedding regardless of the collision system?

Reply:
These settings will eventually influence the resources made available for the reconstruction.
If thresholds are exceeded, you have the tracker skipping the TF.
In the pp case, I would just assume this would just allow for larger combinatorics, which in principle, is just an overestimation.

Reply:
If this is a decision per TF, shouldn't we then set the additional confKey whenever the signal or the background system is PbPb? And not make it depend on embedding?

Reply:
That would cover the cases correctly, yes.